package.json

```diff
@@ -79,6 +79,7 @@
     "chalk": "^2.1.0",
     "commitizen": "^2.9.6",
     "commitlint-config-seek": "^1.0.0",
+    "core-js": "^2.5.3",
```

Comment:
You might want to add this to the greenkeeper ignore list too

Reply:
Yep, nice catch 👍

Reply:
Done.